src/common/mui-theme/components.ts

```diff
@@ -49,20 +49,28 @@ export function getComponents(theme: Theme): ThemeOptions['components'] {
         },
       },
     },
+    MuiInputBase: {
```

Comment:
This was causing the weird styling in inputs. `MuiInputBase` applies the outline an all types of inputs. That's why it's better to define it here and disable in inputs themselves. Otherwise you get double outline.